### PR TITLE
Change in DeckKeyword api - add ParserKeyword *

### PR DESCRIPTION
**Pull request by joakim-hove**
Downstream of: https://github.com/OPM/opm-common/pull/1001

### CODE REVIEW

File: examples/cpchop.cpp [hunk 1]
```diff
@@ -75,6 +76,7 @@ try
     Opm::ParameterGroup param(argc, argv);
     std::string gridfilename = param.get<std::string>("gridfilename");
     Opm::CornerPointChopper ch(gridfilename);
+    Opm::Parser parser;
```
Comment (akva2) [inline examples/cpchop.cpp]: i think it would make sense if this was a chopper internal.

Comment (joakim-hove) [reply examples/cpchop.cpp]: OK - fixed

**Comment by joakim-hove** [comment]
@akva2 : You ok with merging this now?

**Comment by akva2** [comment]
yes

**Pull request merged**